sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseIndexDto.java

```diff
@@ -98,6 +98,8 @@
 	private FollowUpStatus followUpStatus;
 	private Date followUpUntil;
 	private Integer visitCount;
+	private Date changeDate;
```

Comment:
I couldn't find any section in the code where these two new fields are used; are they actually needed?

Reply:
The property is not used.
We need the parameter in the constructor because the field is used in the select part of the query (field has to be in the select part of the query to allow sorting on it, as it is a distinct select).
I can remove the property and keep the parameter with an "unused" annotation.

Reply:
Ah, I see, missed that. Alright!